<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/Clowns_Password.ico</normaloff>../img/Clowns_Password.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="novo_password_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="novo_password_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enviar_nova_senha">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="notificacao">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
